### ISSUE TITLE
different clush.conf options for different hosts

### ISSUE THREAD

**Issue by brianjmurrell** [cea-hpc/clustershell#407]
How can I have different options for different groups of nodes a la `clush.conf` style?

For example for hosts foo-[1-5], I want to set `ssh_options` to one set of options but for foo-[6-9], I want `ssh_options` to be a different set of options.

Is this possible in any way?  `clush.conf` appears to be a global set of options for all nodes.

**Comment by degremont** [comment]
This is not possible inside clustershell but the best way to do this for you is probably to put them in `ssh_config` file.

If you don't want to change your global config, you can specify that in a dedicated `ssh_config` and specify it to be used by clustershell, either on the command line with `"-o -F <FILE>"` or in `clush.conf` with the same options.

**Comment by brianjmurrell** [comment]
I suppose `ssh_options` was a bad example.

What if I want a different `ssh_path` for different groups of hosts?

**Comment by degremont** [comment]
Indeed this is not possible with the command line tools. You can only do that with the python API.

**Comment by brianjmurrell** [comment]
But it's using the `clush` to replace `pdsh` that is my use-case.  :-(

**Comment by degremont** [comment]
This is possible with `pdsh`? Could you point me to a doc explaining how `pdsh` is doing that?

**Comment by brianjmurrell** [comment]
I wasn't saying it was possible with `pdsh`.  I was looking to `clush` as a replacement, to be better than `pdsh` but unfortunately this is one area where it isn't.

**Comment by degremont** [comment]
We could consider this as an RFE, but I assume this is not a blocker for your replacement, as pdsh does not have this feature either. 
And CS has already a lot of very interesting additional features compared to pdsh ;)

**Comment by brianjmurrell** [comment]
No, not a blocker and agreed, that CS does already have lots of interesting features over pdsh.
